res/database/Default/user/scr0llbaer.xml

```diff
@@ -357,9 +357,9 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<job index="2" function="8" required="1" />
 				<job index="3" function="128" required="1" />
 				<job index="4" function="2048" gender="2" required="1" />
-				<job index="5" function="64" required="0" />
-				<job index="6" function="8" required="0" />
-				<job index="7" function="16" required="0" />
+				<!--job index="5" function="64" required="0" /-->
```

Comment:
castable jobs (see https://github.com/TVTower/TVTower/blob/4b6f3629da53ef4e8f840b29bea079c39aef2cb0/source/game.gameconstants.bmx#L1976)
with flag 1-128 can be used as optional and should not always appear.
If they do, please open an ticket.

Reply:
Ok I haven't checked if they always appear. My understanding was that they would always appear, but the player wouldn't be required to cast the role, but apparently that's not how it works.

Reply:
Script templates are templates to create a script. The required-flag indicates if a job must appear in the final script or if it is optional.